Makefile

```diff
@@ -25,3 +25,10 @@ docker-build:
 docker-run:
 	docker run --rm -it -p 12111-12112:12111-12112 "$(IMAGE_NAME):latest"
 .PHONY: docker-run
+
+update-openapi-spec:
+	rm -f ./openapi/spec3.json
+	rm -f ./openapi/fixtures3.json
+	wget https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.json -P ./openapi
+	wget https://raw.githubusercontent.com/stripe/openapi/master/openapi/fixtures3.json -P ./openapi
```

Comment:
I think it might be nice, but is it something we actively use? We seem to live just fine without this feature in the CLI repo.

Reply:
Yeah I think we can start without this for now since checked with CLI and this hasn't been a problem for them.

Just the top-level PR comments to address.